argocon-demos/2023-migrating-from-jenkins-to-argo-for-cicd/README.md

```diff
@@ -0,0 +1,16 @@
+# Migrating from Jenkins to Argo for CI/CD
```

Comment:
Ah, I just noticed that I think we should rename this to "Migrating CI/CD from Jenkins to Argo"

https://colocatedeventsna2023.sched.com/event/1SJQJ/migrating-cicd-from-jenkins-to-argo-bertrand-quenin-intuit-caelen-urquhart-pipekitio

Reply:
need to rename this folder path, too